presto-tests/src/main/java/com/facebook/presto/tests/sql/SqlExecutionException.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.sql;
+
+public class SqlExecutionException
```

Comment:
Because - in the context of `SqlExcutor`'s intedned use case - it's an error if there was a failure in executing the sql and there's no need to bother all the callers to catch it, since nothing can be done anyway.

I need this instead of SQLException so that I don't pollute SqlExecutor and its users with an unneeded dependency of jdbc.